replicatedhq/ship
# Backup install directory

What I Did
------------

Fixed an issue with un-tarring certain assets when running ship multiple time consecutively.

How I Did it
------------

Have `render` step back up any existing installation directory before executing a plan. This happens after the plan is built.

How to verify it
------------

Run ship multiple times in the same directory. You should see `installer.XXXXX.bak` and `installer` directories

Description for the Changelog
------------

If ship is run twice from the same directory, any previous asset directories generated at `installer/` will be backed up to `installer.TIMESTAMP.bak/`

![](https://rnli.org/-/media/rnli/images/findmynearest/lifeboatstation/42941-helvick-head-b-class-atlantic-85-lifeboat-nicholas-leach-16x9.jpg?w=920&h=518&hash=CCD7B3207D66F21E474634A9CD6484C81743E884)
